lib/interfaces/common.js

```diff
@@ -145,7 +145,16 @@ module.exports = function (suites, context, mocha) {
           throw createUnsupportedError('Pending test forbidden');
         }
         if (typeof opts.fn === 'function') {
-          opts.fn.call(suite);
+          const suiteResult = opts.fn.call(suite);
+          if (suiteResult instanceof Promise) {
+            errors.deprecate(
+              'Suite "' +
```

Comment:
Please add "[mocha] Suite ..." to the message as done in other parts of our code. The origin of a warning is clearer that way.

Reply:
@alcuadrado ping - can you make this change?